src/handlers/graphql.ts

```diff
@@ -1,12 +1,8 @@
 import { ApolloServer, BaseContext } from '@apollo/server';
 import { startServerAndCreateLambdaHandler } from '@as-integrations/aws-lambda';
-import { KeyvAdapter } from '@apollo/utils.keyvadapter';
 import { resolvers, typeDefs } from '../schema';
-import { redisClient } from '../utils/redis';
 import TechnologyModel from '../models/Technology';
```

Comment:
Just realizing you deleted this. Make sure it's fine to leave this here.

Reply:
Yeah I was also thinking to leave this one here, I will restore it

Reply:
Ah! I thought of leaving this as is for the time being and once we start working on the graphQL stuff. I was thinking on refactoring.

Reply:
Seems reasonable - just don't forget otherwise you'll have weird errors you don't understand.